.github/workflows/python-app.yml

```
@@ -12,8 +12,28 @@ on:
     types: [published]
 
 jobs:
+  version:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v3
+        with:
+          ref: 'main'
+          ssh-key: ${{ secrets.SSH_KEY }}
+      - name: Set version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Run script
```

```suggestion
      - name: Update version
```
or something more explicit